Assets/MRTK/SDK/Features/Input/Handlers/TeleportHotSpot.cs

```diff
@@ -20,12 +20,10 @@ public override void OnBeforeFocusChange(FocusEventData eventData)
         {
             base.OnBeforeFocusChange(eventData);
 
-            if (!(eventData.Pointer is TeleportPointer)) { return; }
-
-            IMixedRealityTeleportPointer teleportPointer = eventData.Pointer as IMixedRealityTeleportPointer;
-
-            if (teleportPointer == null)
+            if (!(eventData.Pointer is IMixedRealityTeleportPointer teleportPointer) || teleportPointer.IsNull())
```

Comment:
https://www.thomasclaudiushuber.com/2020/03/12/c-different-ways-to-check-for-null/ suggested using 

"is not" instead of !(x is y) which I think is a bit more readable for not nested conditions.

Reply:
Unfortunately, Unity doesn't yet support `is not` :(

Reply:
When i try, it turns into 

![image](https://user-images.githubusercontent.com/3580640/104393135-90a09e00-54f8-11eb-99c6-e6bfca067797.png)